inc/template-tags.php

```diff
@@ -233,13 +236,15 @@ function visualcomposerstarter_comment( $comment, $args, $depth ) {
 		<div class="comment-wrapper">
 			<footer class="comment-meta">
 				<div class="comment-author vcard">
-					<?php  printf( esc_html__("%s %s says: %s", "visual-composer-starter"), '<cite>'.get_comment_author_link().'</cite>', '<span class="says">', '</span>' ); ?>
+					<?php
+					/* translators: 1: comment author, 2: span opening tag, 3. span closing tag */
+					printf( esc_html__( '%1$s %2$s says: %3$s', 'visual-composer-starter' ), '<cite>' . get_comment_author_link() . '</cite>', '<span class="says">', '</span>' ); ?>
 				</div>
 				<div class="comment-metadata">
 					<a href="<?php echo esc_url( get_comment_link( $comment->comment_ID ) ); ?>">
 						<?php
 						/* translators: 1: date, 2: time */
-						printf( esc_html__( 'On %1$s at %2$s' ), get_comment_date(),  get_comment_time() ); ?>
+						printf( esc_html__( 'On %1$s at %2$s','visual-composer-starter' ), get_comment_date(),  get_comment_time() ); ?>
 					</a>
 					<?php edit_comment_link( __( '(Edit)' ), '  ', '' ); ?>
```

Comment:
In this case yes, " On", "at" should be translatable, also this was suggested by travis